config/core/configmaps/features.yaml

```diff
@@ -40,6 +40,13 @@ data:
     # this example block and unindented to be in the data block
     # to actually change the configuration.
 
+    # Default SecurityContext settings to secure-by-default values
+    # if unset.
+    #
+    # This value will default to "enabled" in a future release,
+    # probably Knative 1.8
+    secure-pod-defaults: "disabled"
```

Comment:
I also ran into an issue where I was getting a validation error on seccompProfile, even after setting this to enabled and pulling in the other patch. Enabling the pod security context feature fixed it, but I wonder if we need to consolidate some of the behaviors... at the very least, maybe using secure defaults should turn on pod security contexts also (though something about that feels a little hacky...)

Reply:
I think you're correct about having this feature enable the other one. That actually seems reasonable to me, and at some point if we make this field default-true or even always-enabled, we can retire the other flag, too, possibly.